pages/manual/manual-gettingstarted-build.md

```diff
@@ -29,6 +29,13 @@ The `--recurse-submodules` is necessary to pull dependencies.
  Use front slashes and no space between modules. Configure again. External modules are developed in the [inviwo modules repository](https://github.com/inviwo/modules).
 4. Hit Generate and open the project in your IDE.
 
+{% include note.html content="
+When using a multi-configuration generator (like Visual Studio and most IDEs) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
```

Comment:
**Recommendation: Set build type to RelWithDebInfo** 
Explanation goes here

Link to how to do it in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2019